sesman/chansrv/clipboard.c

```diff
@@ -372,6 +378,16 @@ clipboard_init(void)
         return 0;
     }
 
+    if (0 != config_read(&g_cfg)) {
```

Comment:
Argh, haven't read this comment yet. I see, that is kind of what i was thinking too, but was not sure about architecture and did not know where in xrdp made more sense to stop the data. Or how to tell if data are inbound/outbound. Will try to wrap my head around it and get back to you.